### DISCUSSION TITLE
vant4 toast组件，消息提示一闪而过，设置duration也不起作用

### DISCUSSION THREAD

**Discussion by Liyimei** [youzan/vant#11589] [Q&A]
运行在Android版chorem浏览器中，调用toast提示组件，所有的类型的消息提示都是一闪而过，设置duration都不起作用，有没有朋友遇到过这样的问题呀？求一个解决方法

**Comment by chenxch** [comment]
官方demo一切正常哦，你这个可以定位问题的信息太少了

**Reply by xuanxuan101** [reply]
官方的demo有源码吗？可以参考下吗？我也遇到这个问题了。

**Reply by chenxch** [reply]
[toast demo](https://github.com/youzan/vant/blob/main/packages/vant/src/toast/demo/index.vue)
就在仓库里面

**Reply by xuanxuan101** [reply]
我是设置duration这个属性不生效，全局定义或者是单独定义都没有效果，怎么回事啊？

**Reply by xuanxuan101** [reply]
是引入的问题吗？我有在考虑这方面，不过我是参照按需引入的方式，这个Toast是单独引入的，不知道是不是这个流程有什么问题？单独引入的时候就是在页面中组件加样式的引入。

**Reply by xuanxuan101** [reply]
![image](https://github.com/youzan/vant/assets/46981182/7c98167e-2eb3-47bd-9bae-d6bfe3fb7f16)

**Reply by xuanxuan101** [reply]
![image](https://github.com/youzan/vant/assets/46981182/2873236f-39de-4cb5-be7f-7c0719b71fc7)

**Reply by xuanxuan101** [reply]
![image](https://github.com/youzan/vant/assets/46981182/b9a9cbbd-ab1a-49a8-84e4-c3ed1ead2656)

**Reply by xuanxuan101** [reply]
showToast组件可以设置duration参数吗？全局或者局部，因为我设置了，并没有起作用。

**Comment by shatian114** [comment]
需要调用
`allowMultipleToast()`
调用之前注意先引入
`import { allowMultipleToast } from 'vant'`
